Stoofvlees.md

```diff
@@ -0,0 +1,50 @@
+### Ingredients
+- 2 grote uien
+- klontjes boter
+- 1 kg rundsvlees (zenuwstuk, in gelijke stukken)
+- peper van de molen
```

Comment:
What is this?

Reply:
It means it's pepper from the table mill.